grand/simulation/antenna/generic.py

```diff
@@ -139,22 +155,41 @@ def fftfreq(n, t):
         if dir_frame is not None:
             # Change the direction to the antenna frame
             if isinstance(direction, BaseRepresentation):
-                direction = dir_frame.realize_frame(direction)
-            direction = direction.transform_to(antenna_frame) # Now compute direction vector data in antenna frame
-            direction = direction.data
+                print("dir pre", direction, dir_frame, antenna_frame)
+                # LWP: this joins direction and dir_frame - not needed if we get rid of astropy
+                if grand_astropy:
+                    direction = dir_frame.realize_frame(direction)
+                else:
+                    E = field.E
```

Comment:
I don't understand this line. I guess it is a mistake? Maybe just `pass` as direction is left unchanged here.

Note that dir_frame should not be needed anymore now if the direction is always assumed to be expressed in the GRAND frame.

Reply:
E is undefined before that, and it is needed just below for calculating Ex, Ey, Ez